test/unit/transactions.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+process.env.NODE_ENV = 'test'
+
+const should = require('should')
+
+const transactions = require('../../src/api/transactions')
+
+describe("calculateTransactionBodiesByteLength()", () => {
+  it("should calculate the bodies length of a transaction", async () => {
+    const lengthObj = { length: 0 }
+    let transaction = {
+      body: '123456789'
+    }
+    transactions.calculateTransactionBodiesByteLength(lengthObj, transaction, new WeakSet())
+    lengthObj.length.should.be.exactly(9)
+  })
+  
+  it("should calculate the bodies length of a transaction with hidden bodies", async () => {
+    const lengthObj = { length: 0 }
+    let transaction = {
+      body: '123456789',
+      arbitrary: {
+        property: {
+          body: '123456789'
+        }
+      }
+    }
+    transactions.calculateTransactionBodiesByteLength(lengthObj, transaction, new WeakSet())
+    lengthObj.length.should.be.exactly(18)
+  })
+  
+  it("should calculate the bodies length of a transaction", async () => {
+    const lengthObj = { length: 0 }
+    let transaction = {}
+    try {
+      transactions.calculateTransactionBodiesByteLength(lengthObj, transaction, new WeakSet())
+      lengthObj.length.should.be.exactly(0)
+    } catch (e) {
+      e.should.not.exist()
```

Comment:
`should.not.exist()` isnt a function of the `e` variable and throws and error. The assert should be `should.not.exist(e)`